src/test/codegen/align-enum.rs

```diff
@@ -9,7 +9,7 @@ pub enum Align64 {
     A(u32),
     B(u32),
 }
-// CHECK: %Align64 = type { [0 x i32], i32, [15 x i32] }
+// CHECK: %Align64 = type { [0 x i64], i64, [7 x i64] }
```

Comment:
SO you're actually moving the field? How do you know that won't cause the field sorting to produce worse results?